src/Task2/Machine.java

```diff
@@ -0,0 +1,26 @@
+package Task2;
+
+public class Machine {
+    private String name;
+    private int year;
+
+    public Machine(String name, int age) {
+        this.name = name;
+        this.year = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+    public int getYear() {
+        return year;
+    }
+
+    public  void begin(){
+        System.out.println(name + " почав роботу");
```

Comment:
What is the point of this method if you override it in each class?

Reply:
Good question, this method makes no sense in this case. I can remove this method from the Machine class or I can not override this method. What do you recommend?

Reply:
I recommend using this method only. As a name you can use a name of the current class. It will be enough

Reply:
Okay, I understood